src/main/webapp/style.css

```diff
@@ -110,10 +117,21 @@ body {}
 }
 
 .login-button {
-  position:absolute;
+  background-color: #e7e7e7; 
+  color: black;
+  border: thick rgba(214, 13, 13, 0.4);
+  padding: 15px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px 2px;
+  cursor: pointer;
+  position:fixed;
```

Comment:
here and elsewhere, make sure the format is `key: value` (with one space after the colon and none before)

Reply:
okay I'll change that and make sure I follow that going forward!